core/storage/user/gae_models.py

```diff
@@ -2313,3 +2313,17 @@ def get_by_auth_id(cls, auth_service, auth_id):
         if auth_service == feconf.AUTH_METHOD_GAE:
             return cls.query(cls.gae_id == auth_id).get()
         return None
+
+    @classmethod
+    def get_all_profiles_by_parent_user_id(cls, parent_user_id):
+        """Fetch all user entries with the given parent_user_id.
+
+        Args:
+            parent_user_id: str. User id of the parent_user whose associated
+                profiles we are querying for.
+
+        Returns:
+            UserAuthDetailsModels. List of UserAuthDetailsModel instances mapped
```

Comment:
The typeinfo should be `list(UserAuthDetailsModel)`.

Reply:
Done.